test/ApcTokenStorageTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace {  
```

Comment:
See this: http://marcelog.github.io/articles/php_mock_global_functions_for_unit_tests_with_phpunit.html